mmv1/templates/terraform/examples/workstation_config_encryption_key.tf.erb

```diff
@@ -42,7 +42,7 @@ resource "google_kms_crypto_key" "default" {
 }
 
 resource "google_service_account" "default" {
-  account_id   = "cloud-workstations-kms"
+  account_id   = "workstations%{random_suffix}"
```

Comment:
I've used `account_id` and set it through the `terraform.yaml` like the other tests do, still confused how this will solve the issue - all the `account_id` entries in other tests are set to 'my-account'... will this not cause the same issue?